Source/JavaScriptCore/parser/Lexer.cpp

```diff
@@ -623,7 +623,7 @@ ALWAYS_INLINE T Lexer<T>::peek(int offset) const
 {
     ASSERT(offset > 0 && offset < 5);
     const T* code = m_code + offset;
-    return (code < m_codeEnd) ? *code : 0;
+    return (code < m_codeEnd) ? *code : T { };
```

Comment:
Nit: Is it more common to use the aggregate constructor `T { }` or the direct constructor `T()`? I definitely see more of the latter but maybe that's out of style now?

Reply:
No idea. I prefer the braces but I'm happy to take style from the consensus of other contributors.